org/cloudfoundry.yml

```diff
@@ -2287,4357 +2287,4 @@ orgs:
-          cpu-entitlement-plugin: admin
-          debugserver: admin
-          deployments-diego: admin
-          deployments-loggregator: admin
-          diego-acceptance: admin
-          diego-ci: admin
-          diego-ci-pools: admin
-          diego-dockerfiles: admin
-          diego-logging-client: admin
-          diego-notes: admin
-          diego-release: admin
-          diego-ssh: admin
-          diego-team: admin
-          diego-upgrade-stability-tests: admin
-          diff-exporter: admin
-          dockerapplifecycle: admin
-          dockerdriver: admin
-          dontpanic: admin
-          durationjson: admin
-          ecrhelper: admin
-          envoy-nginx-release: admin
-          eventhub: admin
-          executor: admin
-          filelock: admin
-          fileserver: admin
-          garden: admin
-          garden-ci: admin
-          garden-integration-tests: admin
-          garden-runc-release: admin
-          garden-wiki: admin
-          garden-windows-ci: admin
-          go-diodes: admin
-          go-log-cache: admin
-          go-loggregator: admin
-          go-metric-registry: admin
-          gorouter: admin
-          groot-windows: admin
-          grootfs: admin
-          guardian: admin
-          healthcheck: admin
-          hydrator: admin
-          idmapper: admin
-          inigo: admin
-          lager: admin
-          localdriver: admin
-          localip: admin
-          locket: admin
-          log-cache-cli: admin
-          log-cache-release: admin
-          loggregator-agent-release: admin
-          loggregator-api: admin
-          loggregator-ci: admin
-          loggregator-release: admin
-          metrics-discovery-release: admin
-          nats-release: admin
-          netplugin-shim: admin
-          networking-oss-deployments: admin
-          noisy-neighbor-nozzle: admin
-          operationq: admin
-          policy_client: admin
-          rep: admin
-          route-emitter: admin
-          route-registrar: admin
-          routing-acceptance-tests: admin
-          routing-api: admin
-          routing-api-cli: admin
-          routing-info: admin
-          routing-perf-release: admin
-          routing-release: admin
-          service-metrics-release: admin
-          silk: admin
-          silk-release: admin
-          statsd-injector-release: admin
-          syslog-ci-private: admin
-          syslog-release: admin
-          system-metrics-release: admin
-          system-metrics-scraper-release: admin
-          systemcerts: admin
-          tlsconfig: admin
-          vizzini: admin
-          volman: admin
-          winc: admin
-          winc-release: admin
-          windows-syslog-release: admin
-          windows-tools-release: admin
-          windows2016fs: admin
-          windows2019fs-release: admin
-          windowsfs-online-release: admin
-          workpool: admin
-      wg-service-management-service-fabrik-approvers:
```

Comment:
Will this be removed and then recreated based on the WG charter? Or deletion will be skipped because it is a group included in a wg charter? I am just concerned there will be a gap where the team will lose access to the repo.

Reply:
wg-service-management-service-fabrik-approvers is a standard github team. It will not be removed and there won't be a gap.
The [org automation](https://github.com/cloudfoundry/community/tree/main/org) generates the standard teams from the WG charters and merges the generated teams into the static config of cloudfoundry.yml. The result is then applied by [peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos).

Reply:
Thanks for the clarification Stephan!